### DISCUSSION TITLE
Student Pack Application

### DISCUSSION THREAD

**Discussion by quanmai04** [community/community#140947] [GitHub Education]
### Hi everyone,

I've got my application approved. How long does it take so that i can use it?
<img width="396" alt="image" src="https://github.com/user-attachments/assets/aaaa2463-2030-4560-8cf0-23cdac7af0de">

**Comment by davevad93** [comment]
Hi @quanmai04 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @quanmai04 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
